client/src/components/Forms/MarkdownInput.tsx

```diff
@@ -20,13 +21,24 @@ export default function MarkdownInput({ historicalCommentSelected }: Props) {
 
   const toggleView = () => {
     setPreviewVisible(!previewVisible);
+    renderComment();
   };
 
   const resetText = () => {
     setPreviewVisible(false);
     setText('');
   };
 
+  const renderComment = () => {
+    if (!text) {
+      setComment('Please leave a comment');
+    } else if (text.length < 30) {
+      setComment('Please leave a detailed comment');
+    } else {
+      setComment(null);
+    }
+  };
```

Comment:
```suggestion
  const renderComment = () => {
    return (
      <Typography.Paragraph type="danger">{!text ? "bla-bla-bla" : "bla-bla-bla-2}</Typography.Paragraph>
  };
```
I meant something like this; it's pseudo code. My main point is that it's important to avoid `nested` ternary operators.